reusable_components_nlp/README.md

```diff
@@ -0,0 +1,989 @@
+Kubeflow's CLI allows us to create a project which will allow us to build the configuration we need to deploy our kubeflow and seldon clusters.
+
+
+```python
+!kfctl init kubeflow-seldon
+!ls kubeflow-seldon
+```
+
+    app.yaml
+
+
+Now we run the following commands to basically launch our Kubeflow cluster with all its components. 
+
+It may take a while to download all the images for Kubeflow so feel free to make yourself a cup of ☕.
+
+If you have a terminal you can see how the containers are created in real-time by running `kubectl get pods -n kubeflow -w`.
+
+
+```bash
+%%bash
+cd kubeflow-seldon
+kfctl generate all -V
+kfctl apply all -V
+```
+
+### Now let's run Seldon 
+For this we'll need Helm to be running, so we'll initialise it.
+
+
+```bash
+%%bash
+helm init 
+kubectl rollout status deploy/tiller-deploy -n kube-system
+```
+
+    $HELM_HOME has been configured at /home/alejandro/.helm.
+    Warning: Tiller is already installed in the cluster.
+    (Use --client-only to suppress this message, or --upgrade to upgrade Tiller to the current version.)
+    Happy Helming!
+    deployment "tiller-deploy" successfully rolled out
+
+
+Once it's running we can now run the installation command for Seldon.
+
+As you can see, we are running the Seldon Operator in the Kubeflow namespace. 
+
+
+```python
+!helm install seldon-core-operator --namespace kubeflow --repo https://storage.googleapis.com/seldon-charts
+```
+
+    NAME:   old-mastiff
+    LAST DEPLOYED: Tue May 28 11:47:35 2019
+    NAMESPACE: kubeflow
+    STATUS: DEPLOYED
+    
+    RESOURCES:
+    ==> v1/ClusterRole
+    NAME                          AGE
+    seldon-operator-manager-role  0s
+    
+    ==> v1/ClusterRoleBinding
+    NAME                                 AGE
+    seldon-operator-manager-rolebinding  0s
+    
+    ==> v1/Pod(related)
+    NAME                                  READY  STATUS             RESTARTS  AGE
+    seldon-operator-controller-manager-0  0/1    ContainerCreating  0         0s
+    
+    ==> v1/Secret
+    NAME                                   TYPE    DATA  AGE
+    seldon-operator-webhook-server-secret  Opaque  0     0s
+    
+    ==> v1/Service
+    NAME                                        TYPE       CLUSTER-IP     EXTERNAL-IP  PORT(S)  AGE
+    seldon-operator-controller-manager-service  ClusterIP  10.101.228.36  <none>       443/TCP  0s
+    
+    ==> v1/StatefulSet
+    NAME                                READY  AGE
+    seldon-operator-controller-manager  0/1    0s
+    
+    ==> v1beta1/CustomResourceDefinition
+    NAME                                         AGE
+    seldondeployments.machinelearning.seldon.io  0s
+    
+    
+    NOTES:
+    NOTES: TODO
+    
+    
+
+
+Check all the Seldon Deployment is running
+
+
+```python
+!kubectl get pod -n kubeflow | grep seldon
+```
+
+    seldon-operator-controller-manager-0                       1/1     Running   1          6s
+
+
+### Temporary fix for Argo image
```

Comment:
Please ensure there are outstanding issues for these fixes in Core & Pipelines repos (or wherever the permanent fix belongs)